test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -283,7 +283,7 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 	}
 
 	async toggleSidebar( open = true ) {
-		const sidebarSelector = '.edit-post-sidebar-header';
+		const sidebarSelector = '.interface-complementary-area-header';
```

Comment:
@noahtallen Would it be worth making the tests work with either 7.8 or 7.9? It would mean having a tidy up task to remove the unneeded selector later. But it would also mean we could merge this earlier, rather than whoever ends up shipping 7.9 having to frantically merge this in order to fix everyone's PRs. Plus we'd save a lot of people from having to rebase PRs etc.

```suggestion
		const sidebarSelector = '.edit-post-sidebar-header, .interface-complementary-area-header';
```

`querySelectorAll()` works with the comma operator, so I'm guessing selenium would be ok with it.

Reply:
I'll try it! It would definitely make it easier.